lib/unobtainium/world.rb

```diff
@@ -124,6 +124,14 @@ def driver(label = nil, options = nil)
 
     private
 
+    def clean_chrome_args(options)
+      options[:desired_capabilities]["chromeOptions"]["args"].uniq!
```

Comment:
Here you could do a similar thing.

```ruby
def clean_chrome_args(options)
   new_options = options["desired_capabilities"]["chromeOptions"]["args"].uniq
   new_options[:caps]["chromeOptions"]["args"].uniq!
   new_options
rescue
   options
end
```

I guess it's ok to use the destructive version in ``new_options`` since they're not passed in as an argument.